src/style.css

```diff
@@ -0,0 +1,102 @@
+body {
+  --height: 68px;
+  --height-sm: 40px;
+  --magn-glass-w: 64px;
+  --magn-glass-w-sm: 35px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0,0,0,0) !important;
+  white-space: nowrap !important;
+}
+
+.search-bar {
+  width: var(--bar-w);
+  height: var(--height);
+  margin-top: 20px;
+  border: 1px solid gainsboro;
+  border-radius: 5px;
+  box-shadow: 0 0 4px gainsboro;
+}
+
+.search-bar:hover {
+  box-shadow: 0 2px 5px gainsboro;
+}
+
+.search-bar-sm {
+  height: var(--height-sm);
+}
+
+.search-field {
+  float: right;
+  padding: 0;
+  border: 0;
+  border-radius: 5px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 800;
```

Comment:
It's better to set default style to html tag and use font-family: inherit for inputs